src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -46,6 +48,8 @@ const CheckBoxCustom = (props: {value: boolean; label: string}) => {
             width: 16,
             height: 16
           }}
+          aria-checked={props.value}
+          onValueChange={() => {}}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `aria-checked` attribute is used for accessibility and correctly reflects the current state of the checkbox. However, the `onValueChange` function is empty, which means that the checkbox's value won't change when clicked. This might be intentional if the checkbox is meant to be read-only, but if it's supposed to be interactive, this needs to be addressed.
```diff
-           onValueChange={() => {}}
+           onValueChange={someFunctionToUpdateValue}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->